.circleci/config.yml

```diff
@@ -12,44 +11,40 @@ jobs:
       - restore_cache:
           key: v2-dependencies-{{ checksum "yarn.lock" }}
       - run:
-          name: Install NPM Dependencies
+          name: Install Dependencies
           command: yarn
       - save_cache:
           key: v2-dependencies-{{ checksum "yarn.lock" }}
           paths:
             - node_modules
-            - packages/vanilla/node_modules
-            - packages/react/node_modules
+            - packages/banner/node_modules
```

Comment:
Just read the docs. I'm disappointed to find that these can't be glob patterns.

Reply:
I *know*. Such a bummer.